Source/RapyutaSimulationPlugins/Public/Tools/RRGeneralUtils.h

```diff
@@ -23,6 +23,35 @@ class RAPYUTASIMULATIONPLUGINS_API URRGeneralUtils : public UBlueprintFunctionLi
         World->GetTimerManager().ClearTimer(TimerHandle);
     }
 
+
+    static FTransform RelativeTransform(const FTransform& RefTrans, const FTransform& WorldTrans)
+    {
+        FTransform RelativeTrans;
+
+        FQuat RefQuatInv = RefTrans.GetRotation().Inverse();
+
+        RelativeTrans.SetTranslation(RefQuatInv.RotateVector(WorldTrans.GetTranslation() - RefTrans.GetTranslation()));
+        RelativeTrans.SetRotation(WorldTrans.GetRotation() * RefQuatInv);
+
+        RelativeTrans.NormalizeRotation();
+
+        return RelativeTrans;
+    }
+
+    static FTransform WorldTransform(const FTransform& RefTrans, const FTransform& RelativeTrans)
+    {
+        FTransform WorldTrans;
+        
+        FQuat RefQuat = RefTrans.GetRotation();
+
+        WorldTrans.SetTranslation(RefQuat.RotateVector(RelativeTrans.GetTranslation()) + RefTrans.GetTranslation());
+        WorldTrans.SetRotation(RelativeTrans.GetRotation() * RefQuat);
+
+        WorldTrans.NormalizeRotation();
+
+        return WorldTrans;
+    }
+
```

Comment:
Might these ones have `Get` prefix? And if I see their purposes right, could we utilize [GetRelativeTransform()](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Private/Math/Transform.cpp#L159) like `RefTransf.GetRelativeTransform(WorldTransf)`?
& `RefTransf * RelativeTransf` for the world transform calculation?

Besides, I'd suggest using `RefTransf/WorldTransf/RelativeTransf` with the trailing `f`, just to differentiate itself from a translation value.